book/views.md

```diff
@@ -537,18 +537,18 @@ function my_module_preprocess_views_view_field(array &$variables) {
 This is such a neat trick to show one field *if it exists* otherwise show another field:
 
 1. Add a field for `field1` and exclude it from display.
-1. Add a field for `field2` and exclude it from display. 
-1. Add a third field for `field2` again
-1. In the rewrite results put the token for `field2`.  Eg. `[colorbox]`
-1. In the `No results behavior`, put the token for `field1` e.g. `[field1]`.
+2. Add a field for `field2` and exclude it from display.
+3. Add a third field for `field2` again
+4. In the rewrite results, put the token for `field2`.  Eg. `[colorbox]`
+5. In the `No results behavior`, put the token for `field1` e.g. `[field1]`.
```

Comment:
ha! FYI Markdown is ok with using 1 for each number - it automatically renumbers for rendering.

Reply:
I saw it somehow worked :) 
Thanks for the info